README.md

```diff
@@ -59,16 +59,39 @@ Here is a list of goals in decreasing order of importance:
 * [i3](https://github.com/i3/i3)
 * [RapidJSON](https://github.com/Tencent/rapidjson/)
 * Modern C++17 compiler.
+* CMake (optional)
 
 ## Installation ##
 
+### Install via `make`
```

Comment:
Please distinguish official `Make` build from optional `CMake` build for now, since I don't feel confident about reviewing/maintaining it at the moment. 
```suggestion
### Install via `make` (official)
```